Source/webserver/src/main/resources/static/css/main.css

```diff
@@ -30,195 +30,216 @@ header div#logo a {
 }
 
 header div#logo a img {
-	max-width: 100px;
+    max-width: 100px;
 }
 
 header nav {
-	display: flex;
-	flex-direction: column;
-	justify-content: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
 }
 
 header nav button#menu_toggle {
-	display: none;
-	font-size: 3em;
-	padding: 0;
-	margin: 0;
-	border: none;
-	background: transparent;
-	color: white;
-	cursor: pointer;
+    display: none;
+    font-size: 3em;
+    padding: 0;
+    margin: 0;
+    border: none;
+    background: transparent;
+    color: white;
+    cursor: pointer;
 }
 
 header nav ul {
-	margin: 0;
-	list-style-type: none;
-	display: flex;
-	flex-direction: row;
-	font-size: 1.4em;
+    margin: 0;
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+    font-size: 1.4em;
 }
 
 header nav ul li {
 
 }
 
 header nav ul li a {
-	display: block;
-	color: white;
-	text-decoration: none;
-	padding: 0.5em 1em;
+    display: block;
+    color: white;
+    text-decoration: none;
+    padding: 0.5em 1em;
 }
 
 header nav ul li a:hover {
-	background: rgba(0, 0, 0, 0.1);
+    background: rgba(0, 0, 0, 0.1);
 }
 
 header nav ul li a:focus {
-	background: rgba(0, 0, 0, 0.3);
+    background: rgba(0, 0, 0, 0.3);
 }
 
 header nav ul li a:active {
-	background: rgba(0, 0, 0, 0.5);
+    background: rgba(0, 0, 0, 0.5);
 }
 
 main {
-	flex: 1 1 500px;
-	position: relative;
-	background: rgba(0, 0, 0, 0.5);
+    flex: 1 1 100%;
```

Comment:
Vad är anledningen till ändringen? När vi hade 500px här så kunde inte kartan bli mindre än 500px i höjd vilket lät oss garantera att kartan syns även på små skärmar, med 100% kan kartan försvinna så vi bara ser header/footer.

Reply:
Changed to min 500px